wa/workloads/aitutu/uiauto/app/src/main/java/com/arm/wa/uiauto/aitutu/UiAutomation.java

```diff
@@ -60,14 +60,19 @@ public void extractResults() throws Exception {
     public void clearPopups() throws Exception {
         UiSelector selector = new UiSelector();
 
+        UiObject next = mDevice.findObject(selector.textContains("NEXT")
+                                            .className("android.widget.Button"));
+        next.waitForExists(60000);
```

Comment:
60 seconds seems to be a very long timeout for something that is only present on some devices. Is there an extra 0 here?